community/civilized-discussion.md

```diff
@@ -0,0 +1,112 @@
+
+For example, if there’s a topic on cats, but you prefer dogs, don’t create a post saying “cats are rubbish - I like dogs” - instead create a new topic on dogs with your favorite puppy pictures in.
+
+<a name="participate"></a>
+
+## [Your Participation Counts](#participate)
+
+The conversations we have here set the tone for every new arrival. Help us influence the future of this community by choosing to engage in discussions that make this forum an interesting place to be &mdash; and avoiding those that do not.
+
+Discourse provides tools that enable the community to collectively identify the best (and worst) contributions: bookmarks, likes, flags, replies, edits, watching, muting and so forth. Use these tools to improve your own experience, and everyone else’s, too.
+
+Let’s leave our community better than we found it.
+
+<a name="flag-problems"></a>
+
+## [If You See a Problem, Flag It](#flag-problems)
+
+Moderators have special authority; they are responsible for this forum. But so are you. With your help, moderators can be community facilitators, not just janitors or police.
+
+When you see bad behavior, don’t reply. Replying encourages bad behavior by acknowledging it, consumes your energy, and wastes everyone’s time. _Just flag it_. If enough flags accrue, action will be taken, either automatically or by moderator intervention.
+
+In order to maintain our community, moderators reserve the right to remove any content and any user account for any reason at any time. Moderators do not preview new posts; the moderators and site operators take no responsibility for any content posted by the community.
+
+<a name="be-civil"></a>
+
+## [Always Be Civil](#be-civil)
+
+Nothing sabotages a healthy conversation like rudeness:
+
+* Be civil. Don’t post anything that a reasonable person would consider offensive, abusive, or hate speech.
+* Keep it clean. Don’t post anything obscene or sexually explicit.
+* Respect each other. Don’t harass or grief anyone, impersonate people, or expose their private information.
+* Respect our forum. Don’t post spam or otherwise vandalize the forum.
+
+These are not concrete terms with precise definitions &mdash; avoid even the _appearance_ of any of these things. If you’re unsure, ask yourself how you would feel if your post was featured on the front page of a major news site.
+
+This is a public forum, and search engines index these discussions. Keep the language, links, and images safe for family and friends.
+
+<a name="keep-tidy"></a>
+
+## [Keep It Tidy](#keep-tidy)
+
+Make the effort to put things in the right place, so that we can spend more time discussing and less cleaning up. So:
+
+* Don’t start a topic in the wrong category; please read the category definitions.
+* Don’t cross-post the same thing in multiple topics.
+* Don’t post no-content replies.
+* Don’t divert a topic by changing it midstream.
```

Comment:
Discourse has threads within a thread, kinda like a tree. Sometimes that can lead to interwoven conversations. This rule would effectively disallow using that feature. Is that intended?